README.md

```diff
@@ -7,159 +7,152 @@
-  - [How to Checkstyle](#How-to-Checkstyle)
-- [Documentation](#Documentation))
-- [Contributing](#contributing)
-  - [git flow](#git-flow)
-  - [issue flow](#git-flow)
+  - [Production](#Production)
+- [Documentation](#Documentation)
+  - [Git flow](#git-flow)
+  - [Issue flow](#issue-flow)
+  - [Debugging](#Debugging)
+- [Team](#Team)
 - [FAQ](#faq)
-- [Support](#support)
 - [License](#license)
 
 ---
 
 ## Installation
 
-- All the `code` required to get started
-- Images of what it should look like
-
 ### Required to install
-* Python (3.6.3)
-* PostgreSQL (9.5.9)
-* Django (1.11.6)
-* NodeJS (6.11.4)
-* Redis (3.0.6)
-
-### Environment
-environmental variables
-```properties
-spring.datasource.url=${DATASOURCE_URL}
-spring.datasource.username=${DATASOURCE_USER}
-spring.datasource.password=${DATASOURCE_PASSWORD}
-spring.mail.username=${EMAIL_ADDRESS}
-spring.mail.password=${EMAIL_PASSWORD}
-cloud.name=${CLOUD_NAME}
-api.key=${API_KEY}
-api.secret=${API_SECRET}
-```
+* NodeJS (14.15.0)
 
 ### Clone
 
 - Clone this repo to your local machine using `https://github.com/ita-social-projects/what-front`
 
 ### Setup
 
-- If you want more syntax highlighting, format your code like this:
-
-> update and install this package first
-
-```shell
-$ brew update
-$ brew install SOMEREPOproductions
-```
-
-> now install npm and bower packages
-
-```shell
-$ npm install
-$ bower install
-```
-
-- For all the possible languages that support syntax highlithing on GitHub (which is basically all of them), refer <a href="https://github.com/github/linguist/blob/master/lib/linguist/languages.yml" target="_blank">here</a>.
-
+- install npm packages:
+  ```properties
+  npm install
+  ```
 ### How to run local
 
-### How to run Docker
+- open terminal
+- write to start:
+  ```properties
+  npm run dev
+  ```
+- open _http://localhost:8080_ to view it in the browser.
 
----
+### Production
 
-## Usage
-### How to work with swagger UI
-### How to run tests
-### How to Checkstyle
+- To create production build use:
+  ```properties
+  npm run build
+  ```
 
 ---
 
 ## Documentation
 
 ---
 
-## Contributing
-
 ### Git flow
-> To get started...
+
+> We have **master** , **dev** and **feature** branches.  
+>>All **feature** branches must be merged into **[dev](https://github.com/ita-social-projects/what-front/tree/dev)** branch.
+
+![Github flow](https://www.programmersought.com/images/446/b01b2a0649fee64c9ba71fc10a0ef886.png)
+
 #### Step 1
 
-- **Option 1**
-    - 🍴 Fork this repo!
+- **Clone dev**
+  - Open terminal
+  - To clone repo to your local machine write:
+    ```
+    git clone https://github.com/ita-social-projects/what-front.git
+    ```    
 
-- **Option 2**
-    - 👯 Clone this repo to your local machine using `https://github.com/ita-social-projects/SOMEREPO.git`
+- **Create new feature branch**
+  - To create new branch write:
+    ```
+    git branch branch_name
+    ```
+  - Branch name format: `surname-#issue-name_of_issue`
```

Comment:
Please edit like this:
`[surname]-[#_of_issue]-[name_of_issue]`
and add example:
`drozdov-3-project_documentation`